bigtable/tests/instance_admin_integration_test.cc

```diff
@@ -103,6 +103,21 @@ TEST_F(InstanceAdminIntegrationTest, DeleteInstancesTest) {
       IsInstancePresent(instance_admin_->ListInstances(), instance_id));
 }
 
+/// @test Verify that InstanceAdmin::ListClusters works as expected.
+TEST_F(InstanceAdminIntegrationTest, ListClustersTest) {
+  // The emulator does not support cluster operations.
+  if (UsingCloudBigtableEmulator()) {
+    return;
+  }
+
+  // TODO(#418) - create an instance and test that its cluster is returned here.
```

Comment:
Good news!  There is enough of #418 implemented that you can actually fix this problem.  Please create at least one instance (and delete it), the ListInstances integration code should be a good example.